samples/snippets/cloud-client/src/main/java/recaptcha/MigrateKey.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package recaptcha;
+
+// [START recaptcha_enterprise_migrate_site_key]
+
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.recaptchaenterprise.v1.Key;
+import com.google.recaptchaenterprise.v1.KeyName;
+import com.google.recaptchaenterprise.v1.MigrateKeyRequest;
+import java.io.IOException;
+
+public class MigrateKey {
+
+  public static void main(String[] args) throws IOException {
+    String projectId = "project-id";
+    String recaptchaSiteKey = "recaptcha-site-key";
+
+    migrateKey(projectId, recaptchaSiteKey);
+  }
+
+  /**
+   * Migrate a key from reCAPTCHA (non-Enterprise) to reCAPTCHA Enterprise. If you created the key
+   * using Admin console: https://www.google.com/recaptcha/admin/site, then use this API to migrate
+   * to reCAPTCHA Enterprise. For more info, see:
+   * https://cloud.google.com/recaptcha-enterprise/docs/migrate-recaptcha
+   *
+   * @param projectId: Google Cloud Project Id.
+   * @param recaptchaSiteKey: Specify the site key to migrate.
+   */
+  public static void migrateKey(String projectId, String recaptchaSiteKey) throws IOException {
```

Comment:
Is this one tested?

Reply:
Migrate key can't be tested through client libraries as the non-enterprise version of the key needs to be created only through the Admin Console.